webpack.dev.js

```diff
@@ -50,7 +50,7 @@ module.exports = [
     devServer: configureDevServer(port + 1),
     plugins: [
       new webpack.HotModuleReplacementPlugin(),
-      new DashboardPlugin(dashboard.setData),
+      // new DashboardPlugin(dashboard.setData),
     ],
```

Comment:
Turned this off as on my terminals it was always a mess

Reply:
Sounds good to me. that was an experiment. We should remove it forever... ;-)